packages/fiori/cypress/specs/ShellBar.cy.ts

```diff
@@ -0,0 +1,235 @@
+import { html } from "lit";
+import "../../src/ShellBar.js";
+import "@ui5/webcomponents-icons/dist/activities.js";
+import "@ui5/webcomponents-icons/dist/sys-help.js";
+import "@ui5/webcomponents-icons/dist/da.js";
```

Comment:
import all components that are used inside the templates otherwsie you depend on side effects